coffeecard/CoffeeCard.Library/Utils/DateTimeProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
What is the motivation of this class? I'm thinking it's used to setup DI?

Reply:
Calling `DateTime.UtcNow` is a side effect. For testing we need to wrap it, so we can mock the DateTime